endpoint/engineEth.go

```diff
@@ -261,15 +325,23 @@ func (e *EngineEth) asyncSendRawTransaction(txsBytes []byte) (*string, error) {
 
 // syncSendRawTransaction submits a raw transaction to engine synchronously
 func (e *EngineEth) syncSendRawTransaction(txsBytes []byte) (*string, error) {
-	// check transaction data and return error if any issues (like low gas price or gas limit)
-	err := validateRawTransaction(txsBytes, e.Config.EngineConfig)
+
```

Comment:
Maybe we can add a `deprecated` comment on `syncSendRawTransaction` and `asyncSendRawTransaction` methods. What do you think?

Reply:
done